### PR TITLE
doc: add milestone/version details to development process

### PR DESCRIPTION
**Pull request by bbangert**
issue #41 

r? @jrconlin, @pjenvey

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -51,6 +56,13 @@ applied to categorize the issue appropriately.
 Issues being worked on are moved to the **In Progress** column and assigned to
 the person working on the issue.
 
+### Versions
+
+Push repositories use a {major}.{minor}.{patch} version scheme, with
+patch tags released to address emergency issues in a deployed release.
+
```
Comment (jrconlin) [inline docs/development.md:65]: Do we want to specify that minors add features, majors break backs or is that horse fairly well beaten?

**Comment by jrconlin** [comment]
r+ (with some horse meat that may need tenderizing)

File: docs/development.md (hunk 1 above)
Comment (bbangert) [inline docs/development.md:65]: I think it's worth clarifying, I'm going to add that.

**Comment by bbangert** [comment]
I have tenderized some horse meat.

**Pull request merged**